server/src/main/java/io/deephaven/server/runner/DeephavenApiServerInProcessGroovyComponent.java

```diff
@@ -21,6 +22,7 @@
         GroovyConsoleSessionModule.class,
         ServerBuilderInProcessModule.class,
         SessionToExecutionStateModule.class,
+        HealthCheckModule.class,
```

Comment:
it seems irritating to make this a dependency of tests in this way - is it because the server now has a dependency on the health check module instead of vice versa?

Reply:
yes. The server needs to manipulate the HealthStatusManager in it's shutdown callback, so it now has a dependency on the module.

Maybe there's some way to make this slightly cleaner, but this is what I did which works.